XLS-0075d-account-permission-delegation

```diff
@@ -0,0 +1,309 @@
+
+### 2.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`LedgerIndex`| ✔️|`string`|`Hash256`|The unique ID of the ledger object.|
+|`LedgerEntryType`| ✔️|`string`|`UInt16`|The ledger object's type (`AccountPermission`)|
+|`Account`| ✔️|`string`|`AccountID`|The account that wants to authorize another account.|
+|`Authorize`| ✔️|`string`|`AccountID`|The authorized account.|
+|`Permissions`| ✔️|`string`|`STArray`|The transaction permissions that the account has access to.|
+|`OwnerNode`|✔️|`string`|`UInt64`|A hint indicating which page of the sender's owner directory links to this object, in case the directory consists of multiple pages.|
+|`PreviousTxnID`|✔️|`string`|`Hash256`|The identifying hash of the transaction that most recently modified this object.|
+|`PreviousTxnLgrSeqNumber`|✔️|`number`|`UInt32`|The index of the ledger that contains the transaction that most recently modified this object.|
+
+#### 2.1.1. Object ID
+
+The ID of this object will be a hash of the `Account` and `Authorize` fields, combined with the unique space key for `AccountPermission` objects, which will be defined during implementation.
+
+#### 2.1.2. `Permissions`
+
+This field is an array of permissions to delegate to the account, as listed in [XLS-74d, Account Permissions](https://github.com/XRPLF/XRPL-Standards/discussions/217). The array will have a maximum length of 10.
+
+### 2.2. Account Deletion
+
+The `AccountPermission` object is not a [deletion blocker](https://xrpl.org/docs/concepts/accounts/deleting-accounts/#requirements).
+
+## 3. Transaction: `AccountPermissionSet`
+
+This object represents a set of permissions that an account has delegated to another account, and is modeled to be similar to the [`DepositPreauth` transaction type](https://xrpl.org/docs/references/protocol/transactions/types/depositpreauth).
+
+### 3.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`TransactionType`| ✔️|`string`|`UInt16`|The transaction type (`AccountPermissionSet`).|
+|`Account`| ✔️|`string`|`AccountID`|The account that wants to authorize another account.|
+|`Authorize`|✔️|`string`|`AccountID`|The authorized account.|
+|`Permissions`| ✔️|`string`|`STArray`|The transaction permissions that the account has been granted.|
+
+### 3.1.1. `Permissions`
+
+This transaction works slightly differently from the `DepositPreauth` transaction type. Instead of using an `Unauthorize` field, an account is unauthorized by using an empty `Permissions` list.
+
+### 3.2. Failure Conditions
+
+* `Permissions` is too long (the limit is 10), or includes duplicates.
+* Any of the specified permissions are invalid.
+* The account specified in `Authorize` doesn't exist.
+
```

Comment:
If I try to update to the same content as the current settings (no changes), do I get an error? Or will it succeed as with AccountSet?

Reply:
Probably should succeed